serviceinsight/upgrades/support-policy.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Support Policy
+summary: Versions of ServiceInsight that are currently supported
+reviewed: 2022-02-18
+related:
+ - nservicebus/upgrades/release-policy
+ - nservicebus/upgrades/supported-versions
+ - nservicebus/upgrades/supported-platforms
+ - servicecontrol/upgrades/supported-versions
+---
+
+- Major versions are supported for a period of one year after the release of the next major version.
```

Comment:
This is weird because we've never done a major of ServiceInsight. I'm wondering why we'd ever have a breaking change—it's a GUI app completely dependent upon an API that has its own support policy.

I'm actually going to challenge this - I think only the latest version should be supported, period. Until we have an actual reason to release a 2.0 at which point we could revisit the policy.

Reply:
Updated the text to indicate only the latest version is supported.